corehq/apps/ota/views.py

```diff
@@ -64,10 +61,8 @@ def search(request, domain):
     for key, value in criteria.items():
         search_es = search_es.case_property_query(key, value, fuzzy=(key in fuzzies))
     results = search_es.values()
-    if should_use_sql_backend(domain):
-        cases = [get_instance_from_data(CommCareCaseSQLSerializer, result) for result in results]
-    else:
-        cases = [CommCareCase.wrap(flatten_result(result)) for result in results]
+    # Even if it's a SQL domain, we just need to render the results as cases, so CommCareCase.wrap will be fine
+    cases = [CommCareCase.wrap(flatten_result(result)) for result in results]
```

Comment:
[Note][1]d in spec. 

  [1]: https://docs.google.com/document/d/1HyHhyAiuec8FR-j3BJEfB60n82MXJ5JGsLavzW6F4oQ/edit#heading=h.43nwippisb5g